README.md

```diff
@@ -145,6 +145,8 @@ This project has adopted the [Microsoft Open Source Code of Conduct](https://ope
 Reference Papers
 ----------------
 
+Yu Shi, Guolin Ke, Zhuoming Chen, Shuxin Zheng, Tie-Yan Liu. "Quantized Training of Gradient Boosting Decision Trees" ([link](https://papers.nips.cc/paper_files/paper/2022/file/77911ed9e6e864ca1a3d165b2c3cb258-Paper-Conference.pdf)). Advances in Neural Information Processing Systems 35 (NeurIPS 2022), pp. 18822-18833.
```

Comment:
I saw once that linking directly to the PDF was kind of rude, maybe it should point to: https://papers.nips.cc/paper_files/paper/2022/hash/77911ed9e6e864ca1a3d165b2c3cb258-Abstract.html instead? I see the two below do exactly that

Reply:
Totally fine with me.

Updated in https://github.com/microsoft/LightGBM/pull/5923/commits/6e5975864d07f6265fad2f10ac1646ca26f44909.

For my own knowledge... what's rude about it? Just that clicking it could trigger a download instead of opening a tab in your browser?

Reply:
I think I saw it in a tweet from Andrej Karpathy but it was something like that. Some phones automatically download the file, also you may just want to read the abstract before actually going to the paper.

Reply:
iiiinteresting. Ok yeah makes sense to me! Thanks for telling me about that 😊

Reply:
Sorry for necroposting. I accidentally found this thread and think it might be useful to provide a link to detailed answer for those who come here from search engines.

https://physics.meta.stackexchange.com/a/11401

Reply:
It's ok, thanks very much for sharing this!

That answer is fantastic. I just added it to my bookmarked list of things to share in PR reviews, appreciate you sharing it!